filesmanager/tests/test_filesmanager.py

```diff
@@ -192,9 +192,6 @@ def test_sync_content(self):
     def test_delete_content_with_contents(self):
         """
         Check delete content JSON handler with contents.
-        Expected result:
```

Comment:
why are we removing this?

Reply:
Those break the docs step, can I remove all references of the docs step as we don't actually use it anywhere?